src/components/Dropdown/Dropdown.tsx

```diff
@@ -16,8 +16,7 @@ export interface IDropdownState {
 export class Dropdown extends React.Component<IDropdownProps, any> {
```

Comment:
Suggestion: subclass from BaseComponent, then in constructor, provide a list of deprecated props:

```typescript
constructor() {
  super(props, ['isDisabled']);
}
```